### PR TITLE
Sync with default Rails 4.2.0 templates

### PR DESCRIPTION
**Pull request by exalted**
Created a rails app from scratch using `rails new --skip-bundle -d postgresql --force catchup-web` and compared with our current app.

### CODE REVIEW

File: app/assets/javascripts/application.js [hunk 1]
```diff
@@ -2,7 +2,7 @@
 // listed below.
 //
 // Any JavaScript/Coffee file within this directory, lib/assets/javascripts, vendor/assets/javascripts,
-// or vendor/assets/javascripts of plugins, if any, can be referenced here using a relative path.
+// or any plugin's vendor/assets/javascripts directory can be referenced here using a relative path.
```
Comment (as-cii) [inline app/assets/javascripts/application.js]: This was essential :joy:

File: bin/setup [hunk 2]
```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env ruby
```
Comment (as-cii) [inline bin/setup]: This is neat! Nice job, @exalted :hand:

File: app/assets/javascripts/application.js (hunk 1 above)
Comment (exalted) [inline app/assets/javascripts/application.js]: Right? :fire:

File: bin/setup (hunk 2 above)
Comment (exalted) [inline bin/setup]: It reminds me our much more comprehensive script/bootstrap done simpler.

**Comment by as-cii** [comment]
Well done @exalted 

CI seems :green_heart: so........... :shipit: :fire: :fire:

**Pull request merged**